src/Microsoft.AspNetCore.Blazor.Build/targets/RazorCompilation.targets

```diff
@@ -46,10 +45,6 @@
       <Extensions>Blazor-0.1;Blazor.AngleSharp-0.1;$(CustomRazorExtension)</Extensions>
     </RazorConfiguration>
 
-    <RazorExtension Include="Blazor.AngleSharp-0.1">
-      <AssemblyName>Microsoft.AspNetCore.Blazor.AngleSharp</AssemblyName>
-      <AssemblyFilePath>$(_BlazorAngleSharpAssemblyPath)</AssemblyFilePath>
-    </RazorExtension>
```

Comment:
This will still be needed. You need to specify any assemblies you want to load in the Razor compiler.

Reply:
But with another name or?